src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = sourceString
+    .split(';')
+    .filter((line) => line.trim() !== '')
+    .reduce((object, el) => {
+      let [key, value] = el.split(':').map((item) => item.trim());
+
+      key = key.split('').join('').trim();
+      value = value.split('').join('').trim();
```

Comment:
[CODE KNOWLEDGE] - Similarly to the previous comment, `value.split('').join('')` is also redundant and can be removed.